onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1091,10 +1091,21 @@ def aten_conj_physical(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_constant_pad_nd(self: TensorType, pad: INT64, value: float = 0.0) -> TensorType:
+@torch_op("aten::constant_pad_nd", trace_only=True)
+def aten_constant_pad_nd(self: TTensor, pad: Sequence[INT64], value: float = 0.0) -> TTensor:
```

Comment:
What is the type of pad supposed to be? `Sequence[INT64]` looks unlikely. Is it really expected to be a list of tensors (containing a single element)? Or, it is expected to be a tensor? Or, a list of ints?

`INT64` indicates a tensor (of rank 0) of int64 value.

Reply:
should be INT64